CHANGELOG.md

```diff
@@ -3,81 +3,26 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v0.9.1dev - [date]
```

Comment:
I think it is supposed you keep the history of change log, not just add the change log for the new release.